File: ruby/file.rb
file_name = "file.txt";
passage = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n";

f = File.new(file_name, 'w')
f.write(passage)
f.close

f = File.new(file_name, 'a')
0.upto(999) do |x|
    f.write(passage)
end
f.close

File.unlink(file_name)

File: php/file.php
<?php

$fileName = "file.txt";
$passage = "Lorem ipsum dolor sit amet, consectetur adipiscing elit.\n";

file_put_contents($fileName, $passage);
for ($i=0; $i<1000; $i++) {
    file_put_contents($fileName, $passage, FILE_APPEND);
}
unlink($fileName);
